datahub-web-react/src/app/tags/CreateNewTagModal/TagDetailsSection.tsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import styled from 'styled-components';
+import { Input, ColorPicker } from '@components';
+import { TagDetailsProps } from './types';
+
+const SectionContainer = styled.div`
+    margin-bottom: 24px;
+`;
+
+const FormSection = styled.div`
+    margin-bottom: 16px;
+`;
+
+/**
+ * Component for tag name, description, and color selection
+ */
+const TagDetailsSection: React.FC<TagDetailsProps> = ({
```

Comment:
I'd prefer if we define the types TagDetailsProps etc inside the same file where the component is defined. That's the usual pattern. Currently it's defined inside that dedicated types.ts file!